src/ASM/LR/ASMu2Dmx.C

```diff
@@ -353,6 +362,18 @@ bool ASMu2Dmx::integrate (Integrand& integrand,
             if (b != (size_t)geoBasis-1)
               fe.grad(b+1).multiply(dNxdu[b],Jac);
 
+          // Compute Hessian of coordinate mapping and 2nd order derivatives
+          if (use2ndDer) {
+            if (!utl::Hessian(Hess,fe.hess(geoBasis),Jac,Xnod,
+                              d2Nxdu2[geoBasis-1],fe.grad(geoBasis),true))
+              ok = false;
+
+            for (size_t b = 0; b < m_basis.size() && ok; ++b)
+              if ((int)b != geoBasis)
+                ok &= utl::Hessian(Hess,fe.hess(b+1),Jac,Xnod,
```

Comment:
Since the loop is terminated on ok=false, the &= operator is superfluous, just = will suffice. Alternatively, remove the && ok. (Actually the latter makes more sense since this Hessian call will never return false anyway).

Reply:
the && ok is there to avoid doing this on prior error status. removed the ok &=.